service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -316,6 +347,7 @@ private ContentionEntity findContention(ClaimEntity claim, String diagnosticCode
         return contention;
       }
     }
+    log.warn("Could not find contention with this diagnostic code and claim entity.");
```

Comment:
Are you sure you want this warning here? This basic `findContention` method already returns `null`, so the caller should determine if it wants to log a warning, error, or throw an exception. On line 335 (where this method is called), the caller handles the `null` scenario. In looking at these 2 methods, it seems like the warning should be on the method called `ensureContentionExists`, if you want the warning at all.